1-fp-library/src/main/scala/fplibrary/package.scala

```diff
@@ -0,0 +1,25 @@
+package io.github.sentenza.hacktoberfest
+
+package object fplibrary {
+  // private type Thunk[A]                 = () => A
+  // type Description[A]                   = Thunk[A]
+  private type RegularArrow[A, B]       = A => B
+  private type KleisliArrow[A, B, C[_]] = A => C[B]
+
+  // Why are we extending AnyVal? You said performance, why?
+  // val ac = ab `;` bc
+  implicit final class InfixNotationForPointFree[A, B](private val ab: A => B) extends AnyVal {
+    // Alias: In Haskell compose is a simple dot
+    @inline def `.`[C](bc: B => C): A => C = PointFree.compose(ab, bc)
```

Comment:
Couldn't this be simply
```scala
@inline def `.`[C](bc: B => C): A => C = bc.compose(ab)
```
?

Reply:
I'd also be very careful with this choice, since the `.` is already integral part of the language syntax, the risk of creating confusion seems to me too high for the benefit gained.

Also considering that `compose` and `andThen` are already in the stdlib.

Reply:
I guess that it would be a better choice to use `andThen` and avoid the aliasing for dot. Again, this was just left behind as a trace of what I was trying to do.